src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -1078,25 +1077,6 @@ public void OnTemplateRecycled()
 
 		public override string GetAccessibilityInnerText() => Text;
 
-		protected override void OnVerticalContentAlignmentChanged(VerticalAlignment oldVerticalContentAlignment, VerticalAlignment newVerticalContentAlignment)
```

Comment:
Isn't this a breaking change?

Reply:
It shouldn't be, since the method is still the virtual `OnVerticalContentAlignmentChanged` available in `Control`

Reply:
So this particular block was added by you @ajpinedam in https://github.com/unoplatform/uno/commit/40fe5141ca6b94cebe2673175d92ed2054f70e3c. Do you recall why it was needed?

Reply:
@jeromelaban,  @MartinZikmund  reported the issue (https://github.com/unoplatform/uno/issues/3481) because he was having vertical alignment problems on a sample project he built https://github.com/MartinZikmund/messages-multiplatform-uno-sample

Reply:
The original change was made for macOS, we may place this method as specific to macOS for now.

Reply:
@Youssef1313 in practice, I'd think that removing this method may still be a breaking change if someone calls or inherits from it explicitly while using a TextBox inheriting type. It may not be that common though, but best to avoid this particular breaking change by emptying the method and mark it as an upcoming breaking change. That method should also be internal.

Reply:
Yeah I see. Not sure why it wasn't reported by CI